### DISCUSSION TITLE
2022/09/02/idea%E9%85%8D%E7%BD%AEjsp%E5%BC%80%E5%8F%91%E7%8E%AF%E5%A2%83/

### DISCUSSION THREAD

**Discussion by giscus** [OOJ2003/OOJ2003.github.io#18] [Announcements]
# 2022/09/02/idea%E9%85%8D%E7%BD%AEjsp%E5%BC%80%E5%8F%91%E7%8E%AF%E5%A2%83/

愉悦的使用现代工具做java实验

https://ooj2003.github.io/2022/09/02/idea%E9%85%8D%E7%BD%AEjsp%E5%BC%80%E5%8F%91%E7%8E%AF%E5%A2%83/

**Comment by OOJ2003** [comment]
如果你看不到插入的图片，请使用代理工具，github图床+jsdelivr最近貌似墙的比较厉害

**Reply by OOJ2003** [reply]
怎么没人呢
